ts/parts-more/PackedBubbleSeries.ts

```diff
@@ -1853,9 +1854,7 @@ seriesType<Highcharts.PackedBubbleSeries>(
             const isParentNode = point.isParentNode,
                 parentNodeLayout = this.parentNodeLayout;
             if (isParentNode) {
-                parentNodeLayout.setMaxIterations();
-                parentNodeLayout.currentStep = 0;
-                parentNodeLayout.step();
+                parentNodeLayout.restartSimulation();
```

Comment:
Btw, why do we need to restart the simulation? In theory this is the process should be already defined in `dragNodesMixin`:
- if simulations is running, call `layout.resetSimulation()` to extend the running simulation
- if simulation has ended, call `layout.start()` to start new simulation
- if simulation is disabled, call `chart.redraw()` to run all simulations in sync mode

The above `restartSimulation` seems to be unnecessary?

Reply:
We need to restart the simulation to fix the parent node's behavior on drag&drop (without that repulsing didn't work correctly when the simulation has ended). We could use the layout.start() as well but to achieve the goal we don't need the whole logic of the start() method, so I extracted just the necessary parts to the restartSimulation().

Reply:
Ok, I see. Then perhaps we should do this:
- in drag-nodes mixin, instead of using `series.layout` we should use all layouts (check `chart.graphLayoutsStorage`) - look over all available layouts and reset them all
- test `layout.resetSimulation` with all cases (disabled simulation, running simulation and finished simulation)
- `resetSimulation()` should be able to replace current logic for that:
https://github.com/highcharts/highcharts/blob/d1be8b45541bc903e7c3c3cacd010956a3a5db01/ts/modules/networkgraph/draggable-nodes.ts#L138-L156

Reply:
I am totally for this solution. Great idea to put it all in dragNodesMixin.